docs/MAPS.md

```diff
@@ -25,6 +25,8 @@ Specify `TARGET` variable for changing that (e.g. `TARGET=.`). The script runs `
 see `find_generator_tool.sh` script for an algorithm on how it finds it. All temporary files are created
 with `mktemp` and then removed.
 
+If you do not have osmconvert installed in ```$HOME/osmctools/osmconvert```, the script may compile it from source. In this case you need the following boost libraries: Thread, System, Program Options, Filesystem, Date Time, Atomic, Chrono, Serialization. On Ubuntu you may want to install them using ```apt-get install libboost-thread-dev libboost-system-dev libboost-program-options-dev libboost-filesystem-dev libboost-date-time-dev```
```

Comment:
I think boost libraries are not needed for osmconvert. They are needed for osrm project to build maps routing.